### PR TITLE
show givdrop in harvest modal

### PR DESCRIPTION
**Pull request by MohammadPCh**
Fixed:

- Giveth/GIVeconomy#288

@laurenluz, Could you please give me UX writing for this alert?

### CODE REVIEW

**Comment by aminlatifi** [comment]
image

I am eligible to receive givDrop and have not claimed yet. Nothing is shown for me.

**Comment by aminlatifi** [comment]
```
{
    "balance": {
      "balance": "0",
      "givDropClaimed": false,
      "givStaked": "1000000000000000000",
      "id": "0x7e28d11e749fd800d10805f96e0d489b2c71dad6"
    }
  }
  ```

File: src/services/subgraph.tsx [hunk 1]
```diff
@@ -135,6 +138,8 @@ export const fetchBalances = async (
 		const honeyswapLpStaked = BN(info.honeyswapLpStaked || 0);
 		const givStaked = BN(info.givStaked || 0);
 		const allocationCount = Number(info.allocationCount || 0);
+		const givDropClaimed = Boolean(info.givDropClaimed);
+		console.log(`info.givDropClaimed`, network, info.givDropClaimed);
```
Comment (aminlatifi) [inline src/services/subgraph.tsx]: Remove this line please

File: src/components/modals/HarvestAll.tsx [hunk 2]
```diff
@@ -131,6 +134,36 @@ export const HarvestAllModal: FC<IHarvestAllModalProps> = ({
...
+				!currentBalance.givDropClaimed,
+		);
+
+		if (
+			network === config.XDAI_NETWORK_NUMBER &&
+			!currentBalance.givDropClaimed
+		) {
+			fetchAirDropClaimData(address).then(claimData => {
+				if (claimData) {
+					console.log(`claimData`, claimData);
```
Comment (aminlatifi) [inline src/components/modals/HarvestAll.tsx]: Remove this line and other console.log lines please

**Review by aminlatifi — CHANGES_REQUESTED** [review]
It's ok. Just few lines should be removed

File: src/components/modals/HarvestAll.tsx (hunk 2 above)
Comment (MohammadPCh) [reply src/components/modals/HarvestAll.tsx]: they removed in last commit.

**Pull request merged**